RGBLED/CS/MainPage.xaml.cs

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Windows.Devices.Gpio;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Media;
+
+namespace RGBLED
+{
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+
+        private void Page_Loaded(object sender, RoutedEventArgs e)
+        {
+            var gpio = GpioController.GetDefault();
+
+            // Show an error if there is no GPIO controller
+            if (gpio == null)
+            {
+                GpioStatus.Text = "There is no GPIO controller on this device.";
+                return;
+            }
+
+            var deviceModel = GetDeviceModel();
+            // if you are using raspberry pi uncomment below block of code

```

Comment:
Remove commented out code.  This example is for the UpBoard so only mention UpBoard and use UpBoard pins referenced in schematic.

Reply:
Riddhi suggested to comment out the raspberry pi code rather than deleting. Let us know if need to remove?